docs/csharp/language-reference/keywords/using-directive.md

```diff
@@ -1,40 +1,82 @@
 ---
 description: "using directive - C# Reference"
 title: "using directive - C# Reference"
-ms.date: 07/20/2015
+ms.date: 08/19/2021
 f1_keywords: 
   - "using_CSharpKeyword"
 helpviewer_keywords:
   - "using directive [C#]"
 ms.assetid: b42b8e61-5e7e-439c-bb71-370094b44ae8
 ---
-# using directive (C# Reference)
+# using directive
 
-The `using` directive has three uses:
+The `using` directive allows you to use types defined in a namespace without specifying the fully qualified namespace of that type. In its basic form, the `using` directive imports all the types from a single namespace, as shown in the following example:
 
-- To allow the use of types in a namespace so that you do not have to qualify the use of a type in that namespace:
+```csharp
+using System.Text;
+```
+
+You can apply two different modifiers to a `using` directive:
+
+- The `global` modifier has the same effect as adding the same `using` directive to every source file in your project. This modifier was introduced in C# 10.0.
```

Comment:
This is fascinating, and I love it. I did have a question "can you apply multiple modifiers?" But I realized that it is there already... Could we emphasize that a bit more?

For example, what if I wanted `System.Console` to be both `global` and `static`, so that I could call `WriteLine` from anywhere. Maybe even a code sample to show that off?